pkg/config/config.go

```diff
@@ -49,9 +51,10 @@ type config struct {
 	BatchSize                 int    `env:"ZINC_BATCH_SIZE,default=1024"`
 	MaxResults                int    `env:"ZINC_MAX_RESULTS,default=10000"`
 	AggregationTermsSize      int    `env:"ZINC_AGGREGATION_TERMS_SIZE,default=1000"`
-	MaxDocumentSize           int    `env:"ZINC_MAX_DOCUMENT_SIZE,default=1048576"` // Max size for a single document . Default = 1 MB = 1024 * 1024
-	WalSyncInterval           string `env:"ZINC_WAL_SYNC_INTERVAL,default=1s"`      // sync wal to disk, 1s, 10ms
-	WalRedoLogNoSync          bool   `env:"ZINC_WAL_REDOLOG_NO_SYNC,default=false"` // control sync after every write
+	MaxDocumentSize           int    `env:"ZINC_MAX_DOCUMENT_SIZE"`                   // Max size for a single document . Default = 1048576 = ( 1 MB = 1024 * 1024)
+	MaxDocumentSizeHuman      string `env:"ZINC_MAX_DOCUMENT_SIZE_HUMAN,default=1mb"` // Max size for a single document . Default = 1 MB
```

Comment:
use two ENV for user, it not a good idea, you should try use single ENV `ZINC_MAX_DOCUMENT_SIZE`, but support human size, it should be better.

Reply:
I think you not got my point, you should look at here again.

You shouldn't use two variables for one config, you should think about let config support human format. like 

```
MaxDocumentSize   int `env:ZINC_MAX_DOCUMENT_SIZE,default=1mb`
```

So, i said you should consider change `setField`, let it can support human size.

Add two params for one config is not a good idea.

Reply:
Now all you need to configure is a ZINC_MAX_DOCUMENT_SIZE,It is also feasible without a unit.

Reply:
You are right, it can work, but it is not a good implement. We don't like add a `MaxDocumentSizeHuman` for it.

We have multiple size variables, we can't add second variable for all of it.

We need a commonly solution.

Reply:
What do you ok of turning it into a function to get the corresponding value?

Reply:
Can you add my WeChat: yanghengfei, let's discuss on WeChat.